src/main/java/com/google/sps/data/backgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
+/** Interface for the backgroundTaskManager class. */
```

Comment:
Just curious, do you intend to invoke all these functions in one servlet or are you going to have multiple servlets, each scheduled to run separately? I dont know what the right answer is, just curios if you've thought about it.

Reply:
I haven't thought about it much. I guess since we are using the cloud scheduler, everything probably should be in a different servlet to ease things.